static/app/views/issueDetails/streamline/sidebar/seerSection.tsx

```diff
@@ -99,14 +99,25 @@ export default function SeerSection({
 
   const aiConfig = useAiConfig(group, project);
   const issueTypeConfig = getConfigForIssueType(group, project);
+
+  if (!aiConfig.areAiFeaturesAllowed && !aiConfig.hasResources) {
+    return null;
+  }
+
   const showCtaButton =
     aiConfig.needsGenAIConsent ||
     aiConfig.hasAutofix ||
     (aiConfig.hasSummary && aiConfig.hasResources);
 
+  const onlyHasResources =
+    !aiConfig.needsGenAIConsent &&
+    !aiConfig.hasSummary &&
+    !aiConfig.hasAutofix &&
+    aiConfig.hasResources;
+
```

Comment:
what if you have not consented, but hasSummary and hasAutofix are false, and hasResources is true? Shouldn't we still display the Resources regardless of the consent status?

Reply:
this is a limitation with our current UI right, where we don't show the consent component and just a single resource at the same time, will need to do a slight redesign there

Reply:
but some issues don't have summary and autofix, they just have resources. in that case, we show the resources directly in the sidebar. So why should genAIConsent affect that?

(side note, idk if the consent stuff will conflict with Scott's PR)

Reply:
If someone hasn't consented, the button to set up seer will show right, so this will make the title `Seer` in that case and only resources when there isn't

Reply:
My concern is issues like performance issues, where we show this section but do not expect Seer to show up ever. We shouldn't show the setup there either

Reply:
Good point, adding a `issueTypeDoesntHaveSeer`